test/e2e/egress_test.go

```diff
@@ -52,11 +51,15 @@ func TestEgress(t *testing.T) {
 	ac := []configChange{
 		{"Egress", "true", true},
 	}
+
 	if err := data.mutateAntreaConfigMap(cc, ac, true, true); err != nil {
-		t.Fatalf("Failed to enable NetworkPolicyStats feature: %v", err)
+		t.Fatalf("Failed to enable Egress feature: %v", err)
 	}
 
-	t.Run("testEgressClientIP", func(t *testing.T) { testEgressClientIP(t, data) })
+	t.Run("testEgressClientIP", func(t *testing.T) {
+		skipIfNotIPv4Cluster(t)
```

Comment:
nit: would be good to have a small comment explaining why the test is skipped for an IPv6-only cluster

Reply:
Done
I think you should help to take a look at the new testEgressClientIP test case or skip it?@tnqn